crates/solidity/testing/snapshots/bindings_output/modifiers/simple/input.sol

```diff
@@ -0,0 +1,20 @@
+contract FunctionModifier {
+    uint256 public x = 10;
+    bool public locked;
+
+    modifier noReentrancy() {
```

Comment:
same suggestion here + other tests: removing the unrelated defs/refs that we are not explicitly testing, like the contract fields, and `x`/`i` expressions below, since they add noise to the output.

Reply:
In this case those serve a purpose: to test binding _inside_ the modifier function bodies. The body of the function itself can be elided though, as well as the `x` variable.

Reply:
In the `modifiers/with_args` test it's similar, but it also checks that:
- the modifier invocation can access the function parameters
- the modifier body can access the modifier parameters